aten/src/ATen/templates/TensorBody.h

```diff
@@ -661,6 +662,24 @@ class CAFFE2_API Tensor {
   c10::intrusive_ptr<TensorImpl, UndefinedTensorImpl> impl_;
 };
 
+TORCH_API int64_t get_device(Tensor self);
```

Comment:
Pretty sure this should be `CAFFE2_API`

Reply:
as I understand `TORCH_API` is defined as `CAFFE2_API` in WindowsTorchApiMacro.h